### DISCUSSION TITLE
View - Element Count Naming Rules

### DISCUSSION THREAD

**Discussion by jejecrunch** [githru/githru-vscode-ext#64] [View]
#62 에서 파생된 discussion입니다.

보통 어떤 element의 갯수를 나타내는 변수명은 다들 어떻게 작성하시는지 궁금하고, 그것의 배열에 대한 변수명을 작성해야할 때는 어떻게 작성하시는지 궁금합니다.

_by @vgihan_

**Comment by hanseul-lee** [comment]
갯수 관련 변수 중 `num`과 관련해 이전에 문기님과 이야기한 것(https://github.com/githru/githru-vscode-ext/pull/54) 이 있는데 혹시 도움되실까 싶어 링크 남겨드립니다~!
저도 코드 구현하며 궁금했었는데 다른 분들 이야기 듣고 통일하면 좋을 것 같습니다
